src/main/webapp/app/shared/delete-dialog/delete-dialog.component.html

```diff
@@ -29,6 +29,23 @@ <h4 class="modal-title">
     </div>
     <div class="modal-body">
         <p [jhiTranslate]="deleteQuestion" [translateValues]="translateValues">Are you sure you want to delete?</p>
+
+        @if (entitySummaryTitle) {
+            <b [jhiTranslate]="entitySummaryTitle">Summary</b>
```

Comment:
I think we agreed at some point to leave out the text when using jhiTranslate

```suggestion
            <b [jhiTranslate]="entitySummaryTitle"></b>
```